ThunkLibs/Generator/gen.cpp

```diff
@@ -574,8 +597,18 @@ void GenerateThunkLibsAction::EndSourceFileAction() {
                 }
             }
 
+            FunctionParams args = thunk;
+            auto function_to_call = "fexldr_ptr_" + libname + "_" + function_name;
+            if (thunk.is_hostcall) {
+                // Get the host function pointer by casting the last parameter to the correct signature
+                args.param_types.pop_back();
+                function_to_call = "reinterpret_cast<" + thunk.return_type.getAsString() + "(*)(" + format_function_params(args) + ")>(args->a_" + std::to_string(args.param_types.size()) + ")";
+            } else if (thunk.custom_host_impl) {
+                function_to_call = "fexfn_impl_" + libname + "_" + function_name;
```

Comment:
Shouldn't this be
`(thunk.custom_host_impl ? "fexfn_impl_" : "fexldr_ptr_") << libname << "_" << function_name`

To handle impls as well?

Reply:
This is what the original code was doing, but the `custom_host_impl` case is now handled explicitly (whereas the `!custom_host_impl` case is handled at initialization). So I don't think there should be an additional conditional.